backend/src/infra/controller/TransactionController.ts

```diff
@@ -0,0 +1,38 @@
+import { type FastifyInstance, type FastifyPluginAsync } from "fastify";
+import { type Static, Type } from "@sinclair/typebox";
+import { type GroupRepository } from "src/core/repository/GroupRepository";
+import { AddTransactionToGroup } from "src/core/usecase/AddTransactionToGroup";
+
+const AddTransactionInputSchema = Type.Object({
+  groupId: Type.Number(),
+  payerId: Type.Number(),
+  amount: Type.Number(),
+});
+type AddTransactionInput = Static<typeof AddTransactionInputSchema>;
+
+export const TransactionController: FastifyPluginAsync<{
```

Comment:
Since `Transactions` are always dependant on a group (and don't exist outside a `Group`), I think this should be moved inside `GroupController` or at least have url shaped as `/group/:groupId:/transaction/`